bin/cml/runner.js

```diff
@@ -413,6 +413,7 @@ const run = async (opts) => {
   try {
     winston.info(`Preparing workdir ${workdir}...`);
     await fs.mkdir(workdir, { recursive: true });
+    await fs.chmod(workdir, '777');
```

Comment:
pls no 😅 not `777` can it be `776` at least?

Reply:
also, is this supposed to be recursive for all files or just on the dir?

Reply:
Would `700` work? 🤔 Also, recursive `chmod` can be performed with [`chmodr`](https://www.npmjs.com/package/chmodr) or something similar.

Reply:
Indeed at this stage there is nothing inside. So recursive is not needed.
What probably should be done is set the user group to make subsequent files to inherit the permissions, if any?

Reply:
Also this command substitutes another native `chmod` in the GH driiver. As I say here we should add probably the user group.